forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabRepository.java

```diff
@@ -579,9 +579,9 @@ public URI createPullRequestUrl(HostedRepository target, String targetRef, Strin
     @Override
     public void addCollaborator(HostUser user, boolean canPush) {
         var accessLevel = canPush ? "30" : "20";
-        var data = "user_id=" + user.id() + "&access_level=" + accessLevel;
         request.post("members")
-               .body(data)
+                .body("user_id", user.id())
+                .body("access_level", accessLevel)
```

Comment:
Minor: indentation looks off by 1.

Reply:
I fixed it. 

There is no consistent formatting standard in Skara. :-( In just this file, I found examples of having indentation in multiples of four spaces, e.g. lines 638-640, or (like here) trying to align with the dot, e.g. lines 608-612. I'd *really* prefer that we introduce a well-defined formatting standard, and stick to it. (Possibly doing a one-shot formatting of all code after establishing the standard.)